plone/app/upgrade/v30/tests.py

```diff
@@ -661,10 +662,26 @@ def testAddEmailCharsetProperty(self):
             self.assertEqual(self.portal.getProperty('email_charset'), 'utf-8')
 
     def testUpdateMemberSecurity(self):
+        # These properties were removed in Plone 5, so we add them
+        # manually here if needed and check if they are properly updated by the
+        # updateMemberSecurity upgrade step
         pprop = getToolByName(self.portal, 'portal_properties')
+        try:
```

Comment:
try/except without a specific error are usually a bad idea because they will swallow any exception that might occur.